docs/source/developer/llvm_timings.rst

```diff
@@ -0,0 +1,105 @@
+.. _developer-llvm-timings:
+
+====================
+Notes on timing LLVM
+====================
+
+
+Getting LLVM Pass Timings
+-------------------------
+
+The dispatcher stores LLVM pass timings in the metadata under the
+``llvm_pass_timings`` key. The timings info contains details on how much time
```

Comment:
```suggestion
``llvm_pass_timings`` key. The timings information contains details on how much time
```

Reply:
Also wonder if this should be `llvm_pass_times` instead of `llvm_pass_timings` so as to be consistent with respect to the already present `pipeline_times` that describes similar.

Same applies throughout, should it be `s/Timings/Times` ?